api/services/storeService.js

```diff
@@ -0,0 +1,11 @@
+const { getAllstores } = require('../models/storeDao')
+
+const allStores = async () => {
+  const storeService = await getAllstores()
+  return storeService
```

Comment:
```suggestion
  return await getAllstores()
```
service layer에서 처리할 별다른 로직이 없다면 이렇게 하는 것이 훨씬 깔끔하겠네요!